components/style/properties/properties.mako.rs

```diff
@@ -1223,8 +1223,11 @@ impl ToCss for PropertyDeclaration {
             PropertyDeclaration::CSSWideKeyword(_, keyword) => keyword.to_css(dest),
             PropertyDeclaration::WithVariables(_, ref with_variables) => {
                 // https://drafts.csswg.org/css-variables/#variables-in-shorthands
-                if with_variables.from_shorthand.is_none() {
-                    dest.write_str(&*with_variables.css)?
+                match with_variables.from_shorthand {
+                    Some(shorthand) if shorthand.flags().contains(ALIAS_PROPERTY) =>
+                        dest.write_str(&*with_variables.css)?,
```

Comment:
Could you elaborate on why is this right? This happens to serialize `-moz-transform`, of course, but why is that true for all the properties?